syntax/parser.go

```diff
@@ -1644,6 +1644,11 @@ func (p *Parser) doRedirect(s *Stmt) {
 			}
 			p.doHeredocs()
 		}
+	case WordHdoc:
+		if p.lang == LangPOSIX {
+			p.langErr(p.pos, "herestrings", LangBash, LangMirBSDKorn)
```

Comment:
```suggestion
			p.langErr(r.OpPos, "herestrings", LangBash, LangMirBSDKorn)
```

like in my suggestion before :)